ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/PublicServiceCallAuthTests.scala

```diff
@@ -5,88 +5,133 @@ package com.daml.platform.sandbox.auth
 
 import java.time.Duration
 import java.util.UUID
-
 import com.daml.ledger.api.auth.AuthServiceJWTPayload
-
+import com.daml.test.evidence.scalatest.ScalaTestSupport.Implicits._
 import scala.concurrent.Future
 
 trait PublicServiceCallAuthTests extends SecuredServiceCallAuthTests {
 
   protected def serviceCallWithPayload(payload: AuthServiceJWTPayload): Future[Any] =
     serviceCallWithToken(Some(toHeader(payload)))
 
-  it should "deny calls with an expired read-only token" in {
+  it should "deny calls with an expired read-only token" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit an expired read-only token")
```

Comment:
Could you use a noun at the beginning of these sentences both attacks and happy that describe who is the actor here e.g.
"Ledger API client can exploit an expired read-only token"
"Ledger API client can connect with token expiring tomorrow"
etc

Reply:
Sorry, I see you have added it to the attack definition. Ignore

Reply:
But the comment for the happy path still stands